src/processScript/transform.ts

```diff
@@ -633,12 +633,12 @@ export function transform(
 	if (neededSubscriptLets.size) {
 		mainFunction.body.body.unshift(t.variableDeclaration(
 			`let`,
-			[ ...neededSubscriptLets ].map(name => t.variableDeclarator(
+			[ ...neededSubscriptLets.entries() ].map(([name, secLevel]) => t.variableDeclarator(
```

Comment:
this can be simplified
```suggestion
			[ ...neededSubscriptLets ].map(([ name, secLevel ]) => t.variableDeclarator(
```